server/adaptors/integrations/__test__/kibana_backend.test.ts

```diff
@@ -298,6 +300,130 @@ describe('IntegrationsKibanaBackend', () => {
         404
       );
     });
+
+    it('should reject with a 404 if static data is not found', async () => {
+      const templateName = 'template1';
+      const staticPath = 'path/to/static';
+      mockRepository.getIntegration.mockResolvedValue({
+        getStatic: jest.fn().mockResolvedValue({
```

Comment:
An aside comment : I'm happy to see the { ok, error } pattern is working out so well in these files!